relevanceai/dataset_api/dataset.py

```diff
@@ -554,6 +554,20 @@ def cat_fields(documents, field_name):
             self.dataset_id, cat_fields, updating_args={"field_name": vector_name}
         )
 
+    def to_dict(self):
```

Comment:
You will need to add the `orient: str='records'` argument here because users should know what format their dictionary shoudl be in. The alternatives are `records`, `dict` . Just support `records` for now (which is the current format) and if any other value, raise ValueError